ibm/service/scc/data_source_ibm_scc_posture_credentials.go

```diff
@@ -0,0 +1,571 @@
+							Description: "The URL of a page.",
+						},
+					},
+				},
+			},
+			"previous": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The URL of a page.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"href": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The URL of a page.",
+						},
+					},
+				},
+			},
+			"credentials": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The details of a credentials.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": &schema.Schema{
+							Type:        schema.TypeBool,
+							Computed:    true,
+							Description: "Credentials status enabled/disbaled.",
+						},
+						"id": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Credentials ID.",
+						},
+						"type": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Credentials type.",
+						},
+						"name": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Credentials name.",
+						},
+						"description": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Credentials description.",
+						},
+						"display_fields": &schema.Schema{
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "Details the fields on the credential. This will change as per credential type selected.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"ibm_api_key": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The IBM Cloud API Key. This is mandatory for IBM Credential Type.",
+									},
+									"aws_client_id": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "AWS client Id.This is mandatory for AWS Cloud.",
+									},
+									"aws_client_secret": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "AWS client secret.This is mandatory for AWS Cloud.",
+									},
+									"aws_region": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "AWS region.",
+									},
+									"aws_arn": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "AWS arn value.",
+									},
+									"username": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "username of the user.This is mandatory for DataBase, Kerbros,OpenStack Credentials.",
+									},
+									"password": &schema.Schema{
```

Comment:
Can we add Sensitive: true to those which are some sensitive information like passowrd , apikey, seceretid...so on
Just check which are all can be sensitive

Reply:
Code modified as per review comments